index.bs

```diff
@@ -83,7 +74,7 @@ Infrastructure definition {#definition}
 The specification of the Client Hints **infrastructure** is divided between the
 following specifications and proposals:
 
-* IETF [[!CLIENT-HINTS]]
+* IETF [[!RFC8942]]
```

Comment:
LGTM assuming bikeshed is happy

Reply:
Yeah, it was after I did `bikeshed update`. :)